csharp/ql/src/Stubs/make_stubs_nuget.py

```diff
@@ -68,9 +68,13 @@ def write_csproj_prefix(ioWrapper):
     cmd.append(version)
 helpers.run_cmd(cmd)
 
+sdk_version = '5.0.402'
+print("\n* Creating new global.json file and setting SDK to " + sdk_version)
+helpers.run_cmd(['dotnet', 'new', 'globaljson', '--force', '--sdk-version', sdk_version])
```

Comment:
Shouldn't this command somehow mention `projectDirIn`?

Reply:
dotnet looks traverses the directory structure upwards towards the root from the current work dir to find a global.json file.
We need to change the working directory to the tempDir and store the global.json here as we execute dotnet commands against multiple subfolders in the the tempDir.

Reply:
@hvitved : But you are right - without any changes we will start poluting the folder from which the nuget script is executed.